docs/testing/README.md

```diff
@@ -0,0 +1,189 @@
+# Testing
+
+## How to test
+
+Follow the [Guiding Principles of React Testing Library](https://github.com/testing-library/react-testing-library#guiding-principles) when writing tests.
+
+## What to test
+
+### Accessibility
+
+Automated accessibility tests are required for significant component variations and should use existing `toHaveNoAxeViolations()` and `toHaveNoDAPViolations()` Jest matchers.
+
+Use this structure for automated accessibility tests:
+
+```js
```

Comment:
I haven't been able to verify this, but using the `jsx` tag instead of `js` here may give us syntax highlighting for the React components

Reply:
As an update, I verified and we get extra syntax highlighting for React components with `jsx`

Reply:
While you may see like a word or two with an extra color in the short snippet below, the difference is very stark with the longer tab order test (and would be with similarly long snippets that are heavy on `expect`s):

### js highlighting (current)
![Screen Shot 2020-03-04 at 3 49 47 PM](https://user-images.githubusercontent.com/9057921/75926377-10b59c00-5e30-11ea-830a-0f9a5e03df4f.png)

### jsx highlighting
![Screen Shot 2020-03-04 at 3 50 10 PM](https://user-images.githubusercontent.com/9057921/75926400-1b703100-5e30-11ea-8bde-581f9093f17a.png)

Reply:
Ah! I obviously didn't look very far 😆That's kind of odd though - it looks like highlighting gets busted by the time it hits that closing tag... Does wrapping the `(<Button>test footer button</Button>)` in parentheses do anything for it?

Reply:
Unfortunately not, from what I can tell. I'm just going to the files tab in this PR, click on "edit this file", editing it, and then looking at the markdown preview.

![Screen Shot 2020-03-04 at 4 16 18 PM](https://user-images.githubusercontent.com/9057921/75928329-c7ffe200-5e33-11ea-9246-c4ae66c0416b.png)

Reply:
Thanks! Finally got it -

<img width="907" alt="Documentation" src="https://user-images.githubusercontent.com/3846874/75930781-4d52b900-5e6b-11ea-8eba-05bfe7827a52.png">

Reply:
OH oops! Now I see what you meant in your previous comment, about the footer button. 😆 Updated now.